WindowsServerDocs/administration/hybrid-cloud-print/hybrid-cloud-print-deploy.md

```diff
@@ -290,7 +290,7 @@ To enable authenticated communication with the HCP services, we need to create 3
     - Go to `c:\inetpub\wwwroot\MopriaCloudService\Database` directory.
     - Execute the following command to create the database in this directory:
 
-        `sqlite3.exe MopriaDeviceDb.db .read MopriaSQLiteDb.sql`
+        `"sqlite3.exe" MopriaDeviceDb.db ".read MopriaSQLiteDb.sql"`
```

Comment:
I appreciate the use of double quote characters to encapsulate strings, especially those with spaces in them, but is it really required around the `sqlite3.exe` file itself (no spaces)?

Reply:
Hi @illfated , I did not try to run it without the quotes around sqlite3.exe. I assume it would work.

Reply:
Sounds good, feel free to test and see if it will work without the quotes around the .exe file. That would be cleaner, in my opinion. Looking forward to hear about your results. 🙂